doc/source/whatsnew/v0.18.0.txt

```diff
@@ -137,6 +138,7 @@ other anchored offsets like ``MonthBegin`` and ``YearBegin``.
 Other API Changes
 ^^^^^^^^^^^^^^^^^
 
+- memory_usage now includes values in the index (:issue:`11597`)
```

Comment:
``.memory_usage`` and mention that ``.info`` will include the memory used by the index as well